tools/assets-automation/assets-maintenance-tool/Azure.Sdk.Tools.Assets.MaintenanceTool/Program.cs

```diff
@@ -0,0 +1,83 @@
+using System.CommandLine;
+using Azure.Sdk.Tools.Assets.MaintenanceTool.Model;
+using Azure.Sdk.Tools.Assets.MaintenanceTool.Options;
+using Azure.Sdk.Tools.Assets.MaintenanceTool.Scan;
+
+namespace Azure.Sdk.Tools.Assets.MaintenanceTool
+{
+    public class Program
+    {
+        public static string[] StoredArgs = new string[] { };
+
+        // working directory is where we will look for previous results "output.json"
+
+        // all should honor
+            // --ScanData <-- results from previous scans (todo, currently checks working directory)
+            
+        // SCAN
+            // --configuration: -> path to file
+
+        // BACKUP
+            // --configuration provided?
+                // SCAN
+                // BACKUP
+                // as each tag is backed up, it is saved with suffix _backup
+
+        // RESTORE
+            // --input-tag <tag that has been stored away>
+
+        // CLEANUP
+            // --configuration provided?
+                // SCAN
+                // BACKUP
+                // CLEANUP
+                    // each tag as found by configuration
+
+            // --input-tag <tag on repo>?
+                // SCAN, BACKUP, and CLEANUP individual tag
+        public static void Main(string[] args)
```

Comment:
nit: I would add an empty line here, to denote that lines 36 and 37 are part of the above-given nested comment structure, and not the only comments on Main.